src/utils/user/get_last_message_user_name.js

```diff
@@ -0,0 +1,17 @@
+export default function getLastMessageUserName(userObject) {
+  if (!userObject) {
+    return null;
+  }
+
+  const cut = (text) => (text.length > 10 ? text.slice(0, 7) + "..." : text);
```

Comment:
let's move this function outside of `getLastMessageUserName `, so it will be created only once